src/SDKs/MachineLearningCompute/Management.MachineLearningCompute/Microsoft.Azure.Management.MachineLearningCompute.csproj

```diff
@@ -7,9 +7,9 @@
         <PackageId>Microsoft.Azure.Management.MachineLearningCompute</PackageId>
         <Description>Provides developers with a library to create and manage machine learning compute resources.</Description>
         <AssemblyName>Microsoft.Azure.Management.MachineLearningCompute</AssemblyName>
-        <Version>0.2.0</Version>
-        <PackageTags>Microsoft Azure Machine Learning Compute management;</PackageTags>
-        <PackageReleaseNotes>Orchestrator properties and orchestrator service principal are optional since the resource provider will create one if not provided.</PackageReleaseNotes>
+        <Version>0.3.0</Version>
+        <PackageTags>Microsoft Azure Machine Learning Compute management;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```

Comment:
@shutchings please remove all the extra tags that were removed earlier. Those tags does not make any sense.
All these tags are not required for searching your nuget package. Please remove them. We are cleaning these tags across all SDKs.